src/components/NcSelect/NcSelect.vue

```diff
@@ -1186,6 +1186,7 @@ body {
 
 	.vs__selected {
 		height: 32px;
+		margin: 0;
```

Comment:
This causes the selected elements to directly "dock" onto the top border of the select.

Reply:
Made it the same height so it doesn't matter now

Reply:
It does now every selected element has no gap to the others:

before | after
---|---
![Screenshot 2024-07-22 at 15-23-35 Nextcloud Vue Style Guide](https://github.com/user-attachments/assets/c8abec22-32ce-401b-9d3d-59d9ad1424aa)|![Screenshot 2024-07-22 at 15-23-31 Nextcloud Vue Style Guide](https://github.com/user-attachments/assets/0c1de469-d74c-4492-8018-b0bcde4bd555)
![Screenshot 2024-07-22 at 15-23-06 Nextcloud Vue Style Guide](https://github.com/user-attachments/assets/2f69532d-210c-4818-b544-ee695227f2f0)|![Screenshot 2024-07-22 at 15-22-59 Nextcloud Vue Style Guide](https://github.com/user-attachments/assets/8186f8b9-3e89-47d2-aabe-e7a0abf2dfd1)

Reply:
So easiest: Just add back like 2px margin and make the height `--default-clickable-area - 2 * margin-size`.
For example:
```css
height: calc(var(--default-clickable-area) - var(--default-grid-baseline));
margin: calc(var(--default-grid-baseline) / 2);
```

Reply:
Yep, added it!